docs/end-user-topics/dois.rst

```diff
@@ -0,0 +1,89 @@
+
+In addition, Dockstore can discover DOIs created by a Zenodo-GitHub integration.
+
+Choosing which DOI to display
+-----------------------------
+
+When viewing the public page of an entry, the entry's concept DOI and the selected version's DOI are displayed.
+
+.. figure:: /assets/images/docs/show-doi.png
+   :alt: Show DOIs
+
+Dockstore only displays one DOI associated with a entry, even if there is more than one DOI, e.g., one generated by GitHub-Zenodo integration, and another generated by Dockstore.
+If this situation arises, you can control which DOI is displayed via your dashboard. Select the entry, then click "Manage DOIs", and you'll get a dialog like this. Select your choice
+and click "Save".
+
+   
+.. figure:: /assets/images/docs/manage-dois.png
+   :alt: Manage DOIs
+
+
+Features of the different ways of issuing DOIs
+----------------------------------------------
+
+With the different ways Dockstore supports DOIs, it may be confusing as to how to generate DOIs for your Dockstore entries.  Here's a high level overview of some of the features of each.
+
+
+Creating Snapshots & Requesting DOIs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See :doc:`Creating Snapshots & Requesting DOIs </advanced-topics/snapshot-and-doi>` for more details
+
+* You must initiate the DOI creation for each version
+    * More control over when DOIs are created
+    * You have to individually do it for every tag
+* Version on Dockstore is snapshotted (frozen)
+* Requires a Zenodo account that you link to your Dockstore account
+
+Automatic Dockstore DOI Generation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+    This feature is currently only available on request. Please contact the Dockstore team at `discuss.dockstore.org <https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506>`__ if you want to participate.
+
+* Automatically mints DOIs for all Git tags pushed to GitHub repositories that have a corresponding entry in Dockstore
+* The DOIs are minted in the Zenodo Dockstore community
+* No Zenodo account is required to mint DOIs
+* A linked Zenodo account is required to edit DOIs
+
+GitHub-Zenodo Generation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Zenodo has a feature where you can link your GitHub account, then specify repositories where Zenodo automatically mints DOIs when GitHub releases are created. Note that a GitHub release is not the
+same as a Git tag; GitHub releases require Git tags, but have extra features.
+
+To set up and configure your Zenodo-GitHub integration, navigate to `Zenodo GitHub account settings <https://zenodo.org/account/settings/github/>`__.
+
+The DOIs created by this integration follow a certain pattern that Dockstore can detect. Dockstore will poll Zenodo to see if any new DOIs have been created against Dockstore entries' GitHub repositories.
+
+* Automatic
+* Only mints DOIs for GitHub releases
+* Can mint DOIs for GitHub repos without associated workflows in Dockstore
```

Comment:
This is also bolded 
![image](https://github.com/user-attachments/assets/f1df0826-f4b8-4ae9-9688-7a9abb8b3735)